### DISCUSSION TITLE
Create event logs (Reporting) in file

### DISCUSSION THREAD

**Discussion by AlexisPiST** [dani-garcia/vaultwarden#3545] [Q&A]
Hello,

It is possible to export event logs in a file?

I have already set the variables : 

      ORG_EVENTS_ENABLED: 'true'
      EVENTS_DAYS_RETAIN: 31
      LOG_FILE: /data/vaultwarden.log
      USE_SYSLOG: 'true'
      EXTENDED_LOGGING: 'true'

In this log, i just see WS connection, API, GET and POST but not user actions.

These event logs are only stored in the database and can be consulted on the web in the "Reporting" tab of the organization.

My goal is to be able to use these logs to couple them with Fail2Ban for specific actions.

Thank you for your answers!

**Comment by BlackDex** [comment]
Then you will need to create a script your self. We are not going to output this into the log file, or an other file.
If you want something like that, extract the information from the database.
